cs3api4lab/api/cs3_share_api.py

```diff
@@ -59,10 +59,12 @@ def create(self, endpoint, file_path, grantee, idp, role, grantee_type):
             self.log.info("Created share: " + endpoint + file_path + " for " + idp + ":" + grantee)
             self.log.info(create_response)
             return self._map_given_share(create_response.share)
-        if create_response.status.code == cs3_code.CODE_INTERNAL:
-            raise ShareAlreadyExistsError("Error creating share: "
-                                          + endpoint + file_path
-                                          + " for " + idp + ":" + grantee)
+        elif create_response.status.code == cs3_code.CODE_INTERNAL:
+            self.log.error("Share already exists for file: " + endpoint + file_path + " for " + idp + ":" + grantee)
+            raise ShareAlreadyExistsError("Share already exists for file: " + endpoint + file_path + " for " + idp + ":" + grantee)
+        else:
+            self.log.error("Error creating share: " + endpoint + file_path + " for " + idp + ":" + grantee)
+            raise ShareError("Error creating share: " + endpoint + file_path + " for " + idp + ":" + grantee)
```

Comment:
These errors are shown to users right? I don't like exposing endpoints and idps and such. Can't we have a cleaner message?